velox/tool/trace/OperatorReplayerBase.cpp

```diff
@@ -31,17 +33,46 @@ DEFINE_string(
     task_id,
     "",
     "Specify the target task id, if empty, show the summary of all the traced query task.");
+DEFINE_string(node_id, "", "Specify the target node id.");
```

Comment:
Do we still need these flags?

Reply:
Yes, there may be multiple traced node dir in $rootDir/$taskId.